libs/jsonforms-components/src/lib/common/Ajv.ts

```diff
@@ -4,4 +4,3 @@ export const ajv = new Ajv({ allErrors: true, verbose: true, strict: 'log' });
 
 //Example format: urn:ads:platform:file-service:v1:/files/f6de737e-c5fc-42fe-963b-28bfe14597c4
 ajv.addFormat('file-urn', /^urn:ads:platform:file-service:v[0-9]:\/files\/[a-zA-Z0-9.-]*$/);
-addFormats(ajv);
```

Comment:
@solittlework can you confirm this is really what you want to remove?  this may break the form app because i think this is the Ajv object that the form app uses.

Reply:
Hi @thdlam, that is a good question. The key issue here is that we are not expected to use this ajv instance. The jsonform comes with an internal instance. We might need to find a way to add it to the internal one.